madprops/serializers.py

```diff
@@ -1,8 +1,9 @@
 import json
+import sys
 
 from django.db.models import ForeignKey
 from django.utils.functional import cached_property
-from rest_framework.serializers import ListSerializer, ModelSerializer
+from rest_framework.serializers import ListSerializer, ModelSerializer, BaseSerializer
```

Comment:
I101 Imported names are in the wrong order. Should be BaseSerializer, ListSerializer, ModelSerializer